### ISSUE TITLE
Input props are not refreshed if updated in parent component

### ISSUE THREAD

**Issue by Ealdst** [darrenjennings/vue-autosuggest#142]
- `vue-autosuggest` version: 2.0.2
- `node` version: 8.15.0
- `npm` (or `yarn`) version: 6.4.1

Relevant code or config

```javascript
<template>
  ...
        <vue-autosuggest
          :suggestions="suggestions"
          @input="getSuggestions"
          @selected="searchOnSelect"
          :get-suggestion-value="getSuggestionValue"
          :input-props="{id:'autosuggest-input', placeholder:getInputPlaceholder}"
          v-model="searchPhrase"
        >
        </vue-autosuggest>
...

</template>

<script>
import { VueAutosuggest } from 'vue-autosuggest';

export default {
  name: 'SearchBar',
  components: {
    VueAutosuggest,
  },
  data() {
    return {
      searchType: 'a',
      searchPhrase: '',
      suggestions: [],
    };
  },
  computed: {
    getInputPlaceholder() {
      if (this.searchType === 'a') {
        return this.$t('searchbar.label.placeholderAll');
      }
      return this.$t('searchbar.label.placeholderNumber');
    },
  },
...
</script>
```

What you did:

We have an input to select between 2 different search types in one input box and require a different placeholder for each. We have created a computed property for the placeholder value and passed this to the input-props property of the autosuggest component.

What happened:

The placeholder is not updated on changing the selection value:

![image](https://user-images.githubusercontent.com/52031796/62539681-b29ae900-b84d-11e9-9c2c-6c8ccc1254e5.png)

Reproduction repository:

Problem description:

Internal_inputProps appears to only be assigned on component creation.

Suggested solution:

A binding or watch should be added so Internal_inputProps can update if any property is changed in the parent component.

**Comment by darrenjennings** [comment]
This is a very good point, thanks for reporting. I believe this may have been introduced in #130.

**Comment by darrenjennings** [comment]
Note for fix: should make internal_inputProps a computed prop, so v-bind'ing is reactive.

**Comment by darrenjennings** [comment]
Fixed in version `2.0.3`

**Comment by Ealdst** [comment]
Perfect, tested and working as expected! Thanks very much for the quick resolution.
